ui/app/src/Views/SilenceList/SilenceView.elm

```diff
@@ -1,9 +1,9 @@
-module Views.Shared.SilenceBase exposing (view)
+module Views.SilenceList.SilenceView exposing (view)
```

Comment:
👍 Perfect, just like the AlertList view and AlertView. Thanks, forgot to change that in my previous PRs.